### ISSUE TITLE
Picture Actions/Slideshow?

### ISSUE THREAD

**Issue by viking2** [bkury/skin.night#18]
If I view photos in the default (Estuary) skin and move the mouse to the left or use the left arrow key, I have a "View options" and "Actions", including slideshow. However, with Night Skin, I get different Options when using the left arrow key, but no Actions menu.
How can I access the Actions Menu?

**Comment by bkury** [comment]
Yes, I haven't even added proper photos/pictures support yet. It's hardly used and there are higher priority stuff to do right now but I'll take a look to see how much work is needed.

**Comment by viking2** [comment]
OK. Slideshow not critical. I can always click Next on the remote....
